osp/serializers/form_feedback.py

```diff
@@ -15,28 +15,25 @@ class FormFeedbackReadSerializer(serializers.ModelSerializer):
     class Meta:
 
         model = FormFeedback
-        fields = [
-            'id', 'user', 'form', 'answers', 'acceptance_status'
-        ]
+        fields = ["id", "user", "form", "answers", "acceptance_status"]
```

Comment:
Replace it with `__all__` if all fields are used. Same for other instances as well

Reply:
sure!